docs/source/rest-api.rst

```diff
@@ -1200,8 +1200,6 @@ Create RegisteredModel
 | ``2.0/preview/mlflow/registered-models/create`` | ``POST``    |
 +-------------------------------------------------+-------------+
 
-.. note::
-    Experimental: This API may change or be removed in a future release without warning.
```

Comment:
Maybe we should revert this because model registry APIs for R were added recently.

Reply:
Agreed!

Reply:
I misunderstood this file is for the R client, but it's not.